### ISSUE TITLE
readme

### ISSUE THREAD

**Issue by strengejacke** [easystats/circus#8]
do we really need this long readme with _all_ code? We have a good documentation, so it's easy to see how models were fitted. I think there might be the day where the "list" becomes too long...

**Comment by DominiqueMakowski** [comment]
True true... what would you recommend? `echo=FALSE` the chunks? Move them somewhere else?

**Comment by DominiqueMakowski** [comment]
On the other hand, this package is currently mostly a "convenience" package for easystats, so I don't expect many people to have its README as a bedside reading...

**Comment by strengejacke** [comment]
> True true... what would you recommend? echo=FALSE the chunks? Move them somewhere else?

Yes, we could do that. I'm not regularly updating the readme anyway (shame on me), but instead I try to keep the docs up to date, so users can always access the code to fit the model via `?circus::model` or the website.
